File: LowLoad.java
public class LowLoad {

    public static void main(String[] args) {

        long sum = 0;
        long n = 984600000L;

        System.out.println("Program starting...");
        
        long start = System.nanoTime();
        System.out.printf("Start time: %.3f\n", start / 1_000_000_000.0);

        for (long i = 1; i <= n; i++) {
            sum += i;
        }

        long end = System.nanoTime();

        System.out.printf("End time: %.3f\n", end / 1_000_000_000.0);

        double elapsedSeconds = (end - start) / 1_000_000_000.0;
        System.out.printf("Time elapsed: %.8f seconds\n", elapsedSeconds);
        System.out.printf("Sum: %d", sum);
        System.out.flush();
    }
}

File: low_load.cs
using System;
using System.Diagnostics;

class LowLoad
{
    static void Main(string[] args)
    {
        long sum = 0;
        long n = 984_600_000;

        Console.WriteLine("Program starting...");

        var start = Stopwatch.GetTimestamp();
        Console.WriteLine($"Start time: {(double)start / Stopwatch.Frequency}");

        var sw = Stopwatch.StartNew();

        for (long i = 1; i <= n; i++)
        {
            sum += i;
        }

        sw.Stop();
        var end = Stopwatch.GetTimestamp();

        Console.WriteLine($"End time: {(double)end / Stopwatch.Frequency}");
        Console.WriteLine($"Time elapsed: {sw.Elapsed.TotalSeconds:F8} seconds");
        Console.WriteLine($"Sum: {sum}");

        Console.Out.Flush();
    }
}
